manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -222,6 +224,9 @@ jobs:
       - name: cf
     update:
       serial: true
+    properties:
+      bbs:
+        url: (( grab meta.bbs.api_location  ))
```

Comment:
BBS only has one process active at a time. The check goes to official bbs url. So both machines are checking active bbs. When you stop one, BBS fails over and updates consul, which in turn updates `bbs.service.cf.internal` record to point to correct VM. To make check fail, you need to stop both bbs instances